src/HtmlOptions/EmailHtmlOptions.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Form\HtmlOptions;
+
+use Yiisoft\Validator\Rule\Email;
+
+final class EmailHtmlOptions implements HtmlOptionsProvider
+{
+    private Email $validator;
+
+    public function __construct(Email $validator)
```

Comment:
Why do we need constructor? Validator instance isn't used anywhere.

Reply:
In [the next PR](https://github.com/yiisoft/form/pull/68) it will be [modified](https://github.com/yiisoft/form/blob/d5e4ed3be24e154cc1fff8ca463b4edb45843f2c/src/HtmlOptions/EmailHtmlOptions.php)